easybuild/easyconfigs/x/xcb-proto/xcb-proto-1.11-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'xcb-proto'
+version = '1.11'
+
+homepage = 'http://xcb.freedesktop.org/'
+description = """The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint,
+latency hiding, direct access to the protocol, improved threading support, and extensibility."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.11'
+versionsuffix = '-%s-%s' % (python, pyver)
+dependencies = [(python, pyver)]
+
+pyshortver = '.'.join(pyver.split('.')[0:2])
```

Comment:
@hajgato: if you're OK with this only working with current framework `develop` (and with future EB v2.7.0 and beyond), you can use the new `%(pyver)s` and `%(pyshortver)s` templates (cfr. https://github.com/hpcugent/easybuild-framework/pull/1595)

```python
version = '1.11'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [('Python', '2.7.11')]
...
sanity_check_paths = {
     'files': ['lib/pkgconfig/xcb-proto.pc'],
     'dirs': ['lib/python%(pyshortver)ss/site-packages/xcbgen'],
 }
```

That cleans up this easyconfig quite a bit...